### DISCUSSION TITLE
More than 1 connection point

### DISCUSSION THREAD

**Discussion by 1Cor134** [bcakmakoglu/vue-flow#419] [Q&A] [answered]
Is it possible to have more than 1 connection point for inputs and outputs?  For instance, could you create a node that has 2 input points?

Is it also possible to change the shape of the nodes?  For instance, to a circle or custom shape?

thanks.

**Comment by bcakmakoglu** [comment, marked as answer]
Sure, you can just create your own custom node with any amount of handles you want.

Check out the [custom node example](https://vueflow.dev/examples/nodes/) to get an idea.

Also read the [guide on nodes](https://vueflow.dev/guide/node.html) to get a better understanding.

**Comment by 1Cor134** [comment]
Thank you, I think I understand now.  Love Vue Flow, great job!
